ProcessLib/HT/Tests.cmake

```diff
@@ -365,7 +365,7 @@ AddTest(
     TESTER vtkdiff
     REQUIREMENTS NOT OGS_USE_MPI
     DIFF_DATA
-    GLOB constraint_bc_1e3_pcs_0_ts_*.vtu T T 1e-14 1e-13
+    GLOB constraint_bc_1e3_pcs_0_ts_*.vtu T T 1e-4 1e-4
```

Comment:
Maybe new reference is due?

Reply:
I haven't any idea why the tolerances have to be so "relaxed". It is only frontend2 that is complaining. Other configurations (docker, windows, mac, my home computer) pass with much tighter tolerances.
I have even to relax the tolerances for the sixth time step even more.

Reply:
I get for `p` at time step 1 difference of "abs maximum norm = [5.427110185059905e+04]"

Reply:
The debug and release builds here yield different results. The above difference is for my release *vs.* the reference files.

Reply:
I can not reproduce the divergence of the temperature values from the reference files. I compiled a (clang) debug build as well as a (clang) release build of ogs on my computer. In both versions the tolerances for the temperature was set to almost the machine precision:
```
GLOB constraint_bc_1e3_pcs_0_ts_*.vtu T T 1e-14 1e-13
```
In both cases (debug/release) the tests pass.